rpc/client.go

```diff
@@ -21,32 +21,39 @@ var (
 
 //Client replica client
 type Client struct {
-	end       chan struct{}
-	requests  chan *Message
-	send      chan *Message
-	responses chan *Message
-	seq       uint32
-	messages  map[uint32]*Message
-	wire      *Wire
-	err       error
+	end             chan struct{}
+	requests        chan *Message
+	send            chan *Message
+	responses       chan *Message
+	seq             uint32
+	messages        map[uint32]*Message
+	wire            *Wire
+	replicaDataAddr string
```

Comment:
Sorry, just noticed. It's improper to call this field `replicaDataAddr`. `peerAddr` should be fine.

Will merge after this got fixed.